persistence/sql/src/test/java/org/infinispan/persistence/sql/AbstractSQLStoreFunctionalTest.java

```diff
@@ -519,6 +545,23 @@ protected void createTable(String cacheName, String tableName, ConnectionFactory
                "sex VARCHAR(255) NOT NULL, " +
                "name VARCHAR(255), " +
                "PRIMARY KEY (sex))";
+      } else if (upperCaseCacheName.equals("TESTNUMERICCOLUMNS")) {
+         if (DB_TYPE.equals(DatabaseType.SQLITE)) {
+            tableCreation = "CREATE TABLE " + tableName + " (" +
+                  "keycolumn INTEGER, " +
+                  "simpleLong INTEGER, " +
+                  "simpleFloat REAL, " +
+                  "simpleDouble REAL, " +
+                  "PRIMARY KEY (keycolumn)) STRICT";
```

Comment:
Does this require the STRICT here? Seems overkill no?

Reply:
It seems not necessary. This was me trying to get correct types from SQLite :face_with_head_bandage: